content/docs/dvclive/get-started.md

```diff
@@ -1,166 +1,196 @@
+runs the script from previous steps and tracks the
+[DVCLive outputs](/doc/dvclive/outputs) as `dvc metrics` and `dvc plots`:
+
+```yaml
+stages:
+  train:
+    cmd: python train.py
+    deps:
+      - train.py
+    metrics:
+      - dvclive.json:
+          cache: false
+    plots:
+      - dvclive/scalars:
+          cache: false
 ```
 
-## Outputs
+### Run DVC Experiments
 
-After you run your training code, you should see the following content in the
-project:
+You can now
+[run experiments](/doc/user-guide/experiment-management/running-experiments)
+with `dvc exp run`:
 
 ```dvc
-$ tree
-├── dvclive
-│   ├── images
-│   │   └── img.png
-│   ├── params.yaml
-│   ├── plots
-│   │   └── roc.json
-│   └── scalars
-│       └── acc.tsv
-└── dvclive.json
+$ dvc exp run
+Running stage 'train':
+> python train.py
+...
 ```
 
-The contents of the `dvclive` folder and `dvclive.json` will vary depending on
-the type of data you have logged and whether you have updated the step value or
-not.
+### Compare and Visualize DVC Experiments
 
-See `Live.log()`, `Live.log_param()`/`Live.log_params()`, `Live.log_image()` and
-`Live.log_plot()` for more details.
+After following the above steps, you have enabled different ways of monitoring
+the training, comparing, and visualizing the experiments:
 
-### Metrics report
+#### DVC CLI
 
-If and when `step` is updated, DVCLive generates or updates a report in
-`dvclive/report.{format}` which will contain all the logged data.
+You can use `dvc exp show` and `dvc plots` to compare and visualize metrics,
+parameters and plots across experiments.
 
-![](/img/dvclive-html.gif)
+<admon type="info" icon="book">
 
-The `format` can be HTML) or Markdown depending on the value of the `report`
-argument passed to [`Live()`](/doc/dvclive/api-reference/live#parameters).
+Learn more in the
+[Comparing Experiments](/doc/user-guide/experiment-management/comparing-experiments)
+and [Visualizing Plots](/doc/user-guide/experiment-management/visualizing-plots)
+pages of the user guide.
 
-<admon type="info">
+</admon>
 
-If you don't update the step number, the report won't be generated unless you
-call `Live.make_report()` directly.
+#### DVC Extension for Visual Studio Code
 
-</admon>
+If you have installed the
+[DVC Extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc),
+you can monitor the training and compare and visualize experiments using the
+[`Experiments`](https://github.com/iterative/vscode-dvc/blob/main/extension/resources/walkthrough/experiments-table.md)
+and
+[`Plots`](https://github.com/iterative/vscode-dvc/blob/main/extension/resources/walkthrough/plots.md)
+views.
+
+![Experiments view](https://github.com/iterative/vscode-dvc/raw/main/extension/resources/walkthrough/images/experiments-table.png)
+![Plots view](https://github.com/iterative/vscode-dvc/raw/main/extension/resources/walkthrough/images/plots-trends.png)
 
-## What next?
+### Share Results
```

Comment:
Seems like this should be an H2 to me (since it's more about Studio than the DVC integration). This would also create an entry in the right side CONTENTS nav.